test/test_common-arcana.rb

```diff
@@ -12,6 +12,15 @@ def setup
     sent_messages.clear
   end
 
+class Settings
```

Comment:
No idea how old these are or if there's a better way. Don't want to invest in the suite unless it becomes more used.

Reply:
❓ Is this what you're looking for https://github.com/rpherbig/dr-scripts/blob/master/test/test_afk.rb#L39 ?

Reply:
YES thanks